README.md

```diff
@@ -1,53 +1,30 @@
-# Subspace Desktop
+# subspace / desktop-farmer
```

Comment:
Why did you revert most of the readme here?

Reply:
oops :D